src/Sulu/Bundle/ContentBundle/Resources/js/views/WebspaceOverview/WebspaceOverview.js

```diff
@@ -47,44 +88,54 @@ class WebspaceOverview extends React.Component<ViewProps> {
     }
 
     render() {
-        // TODO: Load this data dynamically from server!
-        const webspaces: Array<Object> = [
-            {key: 'sulu', name: 'Sulu'},
-            {key: 'sulu_blog', name: 'Sulu Blog'},
-            {key: 'sulu_doc', name: 'Sulu Doc'},
-        ];
-
         return (
             <div className={webspaceOverviewStyles.webspaceOverview}>
-                <div className={webspaceOverviewStyles.webspaceSelect}>
-                    <WebspaceSelect value={this.webspace.get()} onChange={this.handleWebspaceChange}>
-                        {webspaces.map((webspace) => (
-                            <WebspaceSelect.Item key={webspace.key} value={webspace.key}>
-                                {webspace.name}
-                            </WebspaceSelect.Item>
-                        ))}
-                    </WebspaceSelect>
-                </div>
-                <Datagrid
-                    store={this.datagridStore}
-                    adapters={['column_list']}
-                />
+                {!this.webspaces &&
+                    <div>
+                        <Loader />
+                    </div>
+                }
+
+                {!!this.webspaces &&
+                    <Fragment>
+                        <div className={webspaceOverviewStyles.webspaceSelect}>
+                            <WebspaceSelect value={this.webspace.get()} onChange={this.handleWebspaceChange}>
+                                {this.webspaces.map((webspace) => (
+                                    <WebspaceSelect.Item key={webspace.key} value={webspace.key}>
+                                        {webspace.name}
+                                    </WebspaceSelect.Item>
+                                ))}
+                            </WebspaceSelect>
+                        </div>
+                        <Datagrid
+                            store={this.datagridStore}
+                            adapters={['column_list']}
+                        />
+                    </Fragment>
+                }
             </div>
         );
     }
 }
 
 export default withToolbar(WebspaceOverview, function() {
-    // TODO: Load this data dynamically from server!
+    if (!this.selectedWebspace) {
+        return {};
+    }
+
+    const options = this.selectedWebspace.allLocalizations.map(function (localization) {
```

Comment:
Any reason you are not using the `() => ...` shortcut as above here?

Reply:
Especially because you could reduce it to something like this:

```javascript
(localization) => ({
    value: localization.localization,
    label: localization.name,
})
```